node-fs-readfile/cat.js

```diff
@@ -0,0 +1,15 @@
+import { readFile } from 'node:fs/promises';
+
+const files = process.argv.splice('2');
+async function concatFiles() {
+  try {
+    const promiseArray = await Promise.all(
```

Comment:
It's best not to put the data type in the variable name since it makes reading it more awkward. The reader can usually figure out quite easily what type it is. I would find `promises` or `readPromises` more readable.